kontrol-service/engine/template/dev-flow.go

```diff
@@ -1,333 +0,0 @@
-package template
-
-import (
-	"fmt"
-	"sort"
-
-	"github.com/samber/lo"
-	"istio.io/api/networking/v1alpha3"
-	"k8s.io/apimachinery/pkg/util/intstr"
-
-	istioclient "istio.io/client-go/pkg/apis/networking/v1alpha3"
-	apps "k8s.io/api/apps/v1"
-	v1 "k8s.io/api/core/v1"
-	metav1 "k8s.io/apimachinery/pkg/apis/meta/v1"
-
-	"kardinal.kontrol-service/types"
-)
-
-func RenderClusterResources(cluster types.Cluster) types.ClusterResources {
```

Comment:
where is all this code now? is it ok to remove this file? or was it accidentally removed?

Reply:
No, this was dead code (not used )